examples/video/index.js

```diff
@@ -67,4 +86,20 @@ function onToggleMute() {
   muteButton.classList.toggle('muted');
 }
 
+function formatTime (time) {
+
+  time = !time || typeof time !== 'number' || time < 0 ? 0 : time;
+
+  var minutes = Math.floor(time / 60) % 60,
+      seconds = Math.floor(time % 60)
+  ;
```

Comment:
No it's the closing one but I tend to leave it separately in the event I need to add more variables